scripts/base/protocols/ssl/main.zeek

```diff
@@ -116,7 +116,7 @@ export {
 
 	## The default root CA bundle.  By default, the mozilla-ca-list.zeek
 	## script sets this to Mozilla's root CA list.
-	const root_certs: table[string] of string &redef;
+	const root_certs: table[string] of string &redef &docs_omit_value;
```

Comment:
Given this is mostly about zeekygen and it's behavior is instructed by `##!, `##<`, or `##` comments, maybe we could support a magic comment rather than a new attribute?
```
..of string &redef ##@omit-value     # we have @no-test
..of string &redef ##@zeekygen-omit-value
..of string &redef ##@omit-value
..of string &redef ###omit-value
..of string &redef ##!omit-value
..of string &redef #&omit_value
```

@bbannier you have opinions here I'm sure - thoughts?

Reply:
My only skin in this game is that attributes always require special handling in a parser to disambiguate against binary `&`, e.g., the parser used for `zeek-format` [hardcodes all attributes](https://github.com/zeek/tree-sitter-zeek/blob/041d869f6af3fd854dec4e9952e5bdbbfb8cfb9d/grammar.js#L255-L284); for that reason introducing new attributes always introduces churn there.

I really like the magic comments idea (and was unaware of `&no-test`). That would not only not affect parsing elsewhere at all, but also avoid questions around how attributes compose which tend to crop up.